### PR TITLE
Improve cmd doc

### PR DESCRIPTION
**Pull request by christiansiegel**
WIP

### CODE REVIEW

**Comment by christiansiegel** [comment]
Resolves #61

**Comment by niiku** [comment]
Cool! Is there going to be a website soon? :D

**Comment by niiku** [comment]
Oh man, I totally missed this one: https://baloise-incubator.github.io/gitopscli/ how cool is this?? <3

File: docs/commands/add-pr-comment.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+
+The `add-pr-comment` command adds a comment to a pull request. You can also reply to an existing comment by providing the `--parent-id`.
+
+## Example
+```bash
+gitopscli add-pr-comment \
+  --git-provider-url https://bitbucket.baloise.dev \
+  --username $GIT_USERNAME \
+  --password $GIT_PASSWORD \
+  --git-user "GitOps CLI" \
```
Comment (joachimprinzbach) [inline docs/commands/add-pr-comment.md]: no need for git-user and git-email in that case - only api calls are performed

File: docs/commands/deploy.md [hunk 2]
```diff
@@ -1,4 +1,95 @@
...
+
+    updated 2 values in example/values.yaml
+
+    frontend.tag: '1.1.0'
+    backend.tag: '1.1.0'
+```
+
+### Create Pull Request
+
+In some cases you might want to create a pull request for your updates. You can achieve this by adding `--create-pr` to the command. The pull request can be left open of merged directly with `--auto-merge`.
```
Comment (joachimprinzbach) [inline docs/commands/deploy.md]: "or" merged directly

File: docs/commands/add-pr-comment.md (hunk 1 above)
Comment (christiansiegel) [reply docs/commands/add-pr-comment.md]: true. removed both

**Pull request merged**